addons/csw/stringtable.xml

```diff
@@ -562,6 +609,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_m252_tube">
             <English>[CSW] M252 Tube Bag</English>
+            <German>[CSW] M252 Mörser Rohr Tasche</German>
```

Comment:
"Mörserrohrtasche" wäre korrekte Schreibweise, wobei ich das eher mit einer Tasche für den Transport von Rohren assoziieren würde. Vielleicht "Mörserrohr in Tasche"?